packages/web-app-files/src/helpers/resources.js

```diff
@@ -64,7 +64,7 @@ export function buildResource(resource) {
     },
     canDownload: function() {
       // TODO: as soon as we allow folder downloads as archive we want to return `true` here without exceptions
```

Comment:
```suggestion
      // TODO: as soon as we allow folder downloads as archive we want to remove !isFolder from this test
```

Reply:
I agree, I searched for a constants file or mapping in the codebase and couldn't find any.  I don't have the background knowledge on the meaning / mapping to do a refactor for it, but it's a good suggestion for improvement.

Reply:
Yes, refactoring the permissions constants is a separate job - not for this PR!

Reply:
Thanks for the comment suggestion, i've added it in.